system/jlib/jlog.cpp

```diff
@@ -338,14 +338,14 @@ StringBuffer & LogMsg::toStringTable(StringBuffer & out, unsigned fields) const
 
 StringBuffer & LogMsg::toStringTableHead(StringBuffer & out, unsigned fields)
 {
+    if(fields & MSGFIELD_msgID)
```

Comment:
Should this code also be using MsgFieldMap?

Reply:
I was going to go over the log file and clean things up: https://track.hpccsystems.com/browse/HPCC-22512 but I'll go ahead and make the improvements for toStringTableHead